vulnfeeds/cmd/nvd-cve-osv/run_cve_to_osv_generation.sh

```diff
@@ -36,7 +36,7 @@ IN_SCOPE_YEARS=($IN_SCOPE_YEARS)
 for YEAR in "${IN_SCOPE_YEARS[@]}"; do
```

Comment:
While we're here, this addresses a little embarrassing oversight that I'll further tidy up in a followup PR

```suggestion
for (( YEAR = 2002 ; YEAR <= $(date +%Y) ; YEAR++ )); do
```